new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -0,0 +1,690 @@
+        domain1Uid + "-" + MANAGED_SERVER_NAME_BASE, replicaCount, replicasAfterScale,
+        null, null);
+
+    // add the third managed server pod original creation timestamp to the list
+    domain1ManagedServerPodOriginalTimestampList.add(
+        getPodOriginalCreationTimestamp(
+            domain1Uid + "-" + MANAGED_SERVER_NAME_BASE + replicasAfterScale,
+            domain1Namespace));
+
+    // verify scaling domain1 has no impact on domain2
+    logger.info("Checking that domain2 was not changed after domain1 was scaled up");
+    verifyDomain2NotChanged();
+  }
+
+  /**
+   * Restart domain1 and verify there was no impact on domain2.
+   */
+  private void restartDomain1AndVerifyNoImpactOnDomain2() {
+    String domain1AdminServerPodName = domainAdminServerPodNames.get(0);
+
+    // shutdown domain1
+    logger.info("Shutting down domain1");
+    assertTrue(shutdown(domain1Uid, domain1Namespace),
+        String.format("shutdown domain %s in namespace %s failed", domain1Uid, domain1Namespace));
+
+    // verify all the server pods in domain1 were shutdown
+    logger.info("Checking that admin server pod in domain1 was shutdown");
+    checkPodDoesNotExist(domain1AdminServerPodName, domain1Uid, domain1Namespace);
+
+    logger.info("Checking managed server pods in domain1 were shutdown");
+    for (int i = 1; i <= replicasAfterScale; i++) {
+      String domain1ManagedServerPodName = domain1Uid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      checkPodDoesNotExist(domain1ManagedServerPodName, domain1Uid, domain1Namespace);
+    }
+
+    // restart domain1
+    logger.info("Restarting domain1");
+    assertTrue(restart(domain1Uid, domain1Namespace),
+        String.format("restart domain %s in namespace %s failed", domain1Uid, domain1Namespace));
+
+    // verify domain1 is restarted
+    // check domain1 admin server pod exists and ready, also check admin service exists in the domain1 namespace
+    logger.info("Checking admin server pod in domain1 was restarted");
+    checkPodExistsReadyAndServiceExists(domain1AdminServerPodName, domain1Uid, domain1Namespace);
+    checkPodRestarted(domain1Uid, domain1Namespace, domain1AdminServerPodName,
+        domainAdminPodOriginalTimestamps.get(0));
+
+    // check managed server pods in domain1
+    logger.info("Checking managed server pods in domain1 were restarted");
+    for (int i = 1; i <= replicasAfterScale; i++) {
+      String domain1ManagedServerPodName = domain1Uid + "-" + MANAGED_SERVER_NAME_BASE + i;
+      checkPodExistsReadyAndServiceExists(domain1ManagedServerPodName, domain1Uid, domain1Namespace);
+      checkPodRestarted(domain1Uid, domain1Namespace, domain1ManagedServerPodName,
+          domain1ManagedServerPodOriginalTimestampList.get(i - 1));
+    }
+
+    // verify domain 2 was not changed after domain1 was restarted
+    logger.info("Verifying that domain2 was not changed after domain1 was restarted");
+    verifyDomain2NotChanged();
```

Comment:
this has to be done after domain1 was shutdown as well.

Reply:
fixed